sdk-base/src/main/java/com/mapbox/maps/extension/observable/eventdata/CameraChangedEventData.kt

```diff
@@ -0,0 +1,6 @@
+package com.mapbox.maps.extension.observable.eventdata
+
+/**
+ * The data class for camera-changed event data in Observer
+ */
+class CameraChangedEventData
```

Comment:
Added this stub class to hold the camera changed event data, currently there's nothing in this event data, but we could add the data in the future without breaking our API.

We do have the startTime property for all event data, we didn't expose them at the moment.

cc @mapbox/maps-android

Reply:
Actually we should add the `begin` and optional `end` timestamp property to all events, as they are stated in the Event class' documentation.